coalib/bears/Bear.py

```diff
@@ -57,6 +64,8 @@ class Bear(Printer, LogPrinter):
 
     LANGUAGES = ()
 
+    REQUIREMENTS = ()
```

Comment:
as we're on it, I think you can write this directly beneath `LANGUAGES`, so it's a bit more dense :+1: